### DISCUSSION TITLE
[Feature request] Wishlist

### DISCUSSION THREAD

**Discussion by superm1991** [ChristopherKlay/StadiaEnhanced#99] [Ideas & Requests]
Hi!

Could a wishlist system be added to Stadia Enhanced?
It looks to me like it shouldn't be too hard to have a little Heart icon in the Store (similar to the Eye icon of the filter system in the Homescreen).

What I'm looking for:
- A new Store page with only items from your wishlist
- Desktop or chrome notification (or any other kind of notification) if at least one item from whishlist is on sale

Let me know what you think :)

**Comment by ChristopherKlay** [comment]
The current issue is that Stadia isn't featuring a "all items" list, it only has a "all games" list, that excludes a ton of different items. Prices can also only be manually checked right now, unless you are on a store list featuring the item - since there isn't any kind of API and Stadia kicks you out entirely, if you request data to frequently (which is why i had to remove the direct link to achievements).

It should definitely be possible to have wishlisting for games (and iirc bundles) however, since that can use the list above.

I'll be looking into it (:

**Comment by superm1991** [comment]
Hi Christopher,

Any news on this topic?

I was thinking that a good first step would be to add a simple "On wishlist" filter alongside "Games"/"Add-ons"/"Bundles" on all store pages.

What do you think?

**Reply by ChristopherKlay** [reply]
Currently working on a new version and decided to delay it a couple more days, since there isn't any huge bug that needs fixing (the streaming mode only) to push the update from a minor one (1.8.12 with just a couple of fixes and a new feature) to a bigger one (1.9.0 with multiple highly requested features).

If nothing goes wrong, this should be included (:

**Reply by superm1991** [reply]
Awesome!
